### PR TITLE
Added reproducibility test for install_pkgs rule

### PR DESCRIPTION
**Pull request by ArthurRab**
Also updated dependencies, as new rules needed to be used from them for
this.

### CODE REVIEW

File: tests/package_managers/BUILD [hunk 1]
```diff
@@ -120,3 +121,35 @@ rule_test(
...
+download_pkgs(
+    name = "download_pkg_git",
+    image_tar = "//ubuntu:ubuntu_16_0_4_vanilla.tar",
+    packages = [
+        "git",
+    ],
+)
+
+install_pkgs(
+    name = "install_git_for_reproducibility1",
```
Comment (xingao267) [inline tests/package_managers/BUILD]: install_git_for_reproducibility_1

File: tests/package_managers/BUILD [hunk 2]
```diff
@@ -120,3 +121,35 @@ rule_test(
...
+    image_tar = "//ubuntu:ubuntu_16_0_4_vanilla.tar",
+    installables_tar = ":download_pkg_git.tar",
+    output_image_name = "test_install_pkgs_duplicate",
+)
+
+install_pkgs(
+    name = "install_git_for_reproducibility2",
```
Comment (xingao267) [inline tests/package_managers/BUILD]: install_git_for_reproducibility_2

File: tests/package_managers/BUILD [hunk 3]
```diff
@@ -120,3 +121,35 @@ rule_test(
...
+    image_tar = "//ubuntu:ubuntu_16_0_4_vanilla.tar",
+    installables_tar = ":download_pkg_git.tar",
+    output_image_name = "test_install_pkgs_duplicate",
+)
+
+compare_ids_test(
+    name = "install_pkgs_reproducibility_test",
+    tars = [
+        ":install_git_for_reproducibility1",
```
Comment (xingao267) [inline tests/package_managers/BUILD]: not sure if we need to do :install_git_for_reproducibility_1.tar. But if it works, then that's good.

Comment (ArthurRab) [reply tests/package_managers/BUILD]: This works as well, I think because install_pkgs only has 1 output target.

**Pull request merged**